client/app/templates/collection/index.hbs

```diff
@@ -69,7 +69,10 @@
             </div>
         </div>
         <div class="row">
-            {{#each model.list as |item|}}
+            {{#each model.groups as |group|}}
```

Comment:
I am not sure if this is a good approach, because it separates the groups and items. We may need to sort alphabetically instead and the view should be able to allow that. Another approach is to combine them into items in the controller or route.